$NetBSD: patch-ac-buildsystem-includedir,v 1.1 2011/08/11 05:50:37 dholland Exp $

Fix handling of the include directory name so it works with current C
preprocessors.

--- XView.obj	Thu Aug 11 00:52:36 2011 -0400
+++ XView.obj	Thu Aug 11 00:53:26 2011 -0400
@@ -15,7 +15,7 @@
             STD_DEFINES = StandardDefines LibXViewDefines 
 #endif
 HEADERS= ${HDRSPUBLIC} ${HDRSPRIVATE}
-INCLUDES= -I$(BUILDINCDIR)$(INCDIR) 
+INCLUDES= -I$(BUILDINCDIR)/$(INCDIR) 
 OBJS = $(TEXT_OBJS) $(DATA_OBJS) $(SPECIAL_OBJS)
 #if !NotLibXView
 ALLFILES = $(SRCS) $(HEADERS)
@@ -85,14 +85,14 @@
 
 #if SubdirHasPublicHeaders
 # if NotLibXView
-BuildIncludes($(HDRSPUBLIC),$(PUB_INCDIR),/**/)
+BuildIncludes($(HDRSPUBLIC),$(PUB_INCDIR),.)
 
 install::
 	$(MKDIRHIER) $(PUB_HEADER_DEST)
 
 InstallMultiple($(HDRSPUBLIC),$(PUB_HEADER_DEST))
 # else
-BuildIncludes($(HDRSPUBLIC),$(XVPUB_INCDIR),/**/)
+BuildIncludes($(HDRSPUBLIC),$(XVPUB_INCDIR),.)
 
 install::
 	$(MKDIRHIER) $(XVPUB_HEADER_DEST)
@@ -103,9 +103,9 @@
 
 #if SubdirHasPrivateHeaders
 # if NotLibXView
-BuildIncludes($(HDRSPRIVATE),$(PRI_INCDIR),/**/)
+BuildIncludes($(HDRSPRIVATE),$(PRI_INCDIR),.)
 # else
-BuildIncludes($(HDRSPRIVATE),$(XVPRI_INCDIR),/**/)
+BuildIncludes($(HDRSPRIVATE),$(XVPRI_INCDIR),.)
 # endif
 # if InstallPrivateHeaders
 #  if NotLibXView
--- XView.tmpl	Thu Aug 11 00:52:36 2011 -0400
+++ XView.tmpl	Thu Aug 11 00:53:26 2011 -0400
@@ -224,7 +224,7 @@
  *	afterwards.
  */
 #ifndef XVIncludeDirName
-#define XVIncludeDirName /include
+#define XVIncludeDirName include
 #endif
                  INCDIR = XVIncludeDirName
 /*
